megamek/src/megamek/common/TargetRoll.java

```diff
@@ -179,6 +179,11 @@ public void addModifier(int value, String desc) {
         addModifier(new TargetRollModifier(value, desc));
     }
 
+    /**
+     * Does not do anything, necessary because Java does not have a "noop" function.
+     */
+    public void doNothing() {}
```

Comment:
this is one of the downsides of functional programming / lambdas, it needs a `noop` :/